conf.py

```diff
@@ -61,14 +68,18 @@
     extensions = _extensions
 
 
-if not os.environ.get('SPHINX_GETTEXT') == 'True':
+if os.environ.get('SPHINX_GETTEXT') is None:
```

Comment:
Me imagino que está explícito, por si tiene un valor, cual sea.
```suggestion
if not os.environ.get('SPHINX_GETTEXT'):
```

Reply:
Era para hacerlo explícito la verdad, a pesar que tenemos cosas tan raras como:

```py
>>> not None
True
```
era para dejar en claro que quería mirar si `get` retornaba `None`, explicando así que la variable no está definida.

El problema con usar `not os.environ.get(...)` es que si la variable está definida a nada, entrará, y la idea es que si alguien ya la definió por alguna razón, que no entre. Por ejemplo:

```py
>>> import os
>>> r = os.environ.get("LALA")
>>> r
>>> r is None
True
>>> 

% export LALA=""
% export LELE=0

% python
>>> import os
>>> os.environ.get("LALA")
''
>>> os.environ.get("LELE")
'0'
>>> not os.environ.get("LALA")
True
>>> not os.environ.get("LELE")
False
>>> not os.environ.get("ALGO_QUE_NO_EXISTE")
True
```

pero para términos de este script, dudo que en algún momento alguien defina la variable a algo más :P pero es la costumbre de estar preparado para todo jaja

Reply:
Está bien chistoso porque justo el `0` arroja un False jajaja.
Igual pues, con que te regrese un '' o no esté (None) daría True, pero sí, igual y un poco exagerado, el `is None` está bastante explícito, bueno saberlo la verdad.